docs/guides/qpu-information.mdx

```diff
@@ -37,75 +41,65 @@ The revision version number will increment for fixes that do not break the exist
 *   Small electronics changes that don’t affect operation.
 *   QPU software updates.
 
-## QPU configuration values
+## QPU details
 
-The following is a subset of QPU configuration values available in IBM Quantum Platform and from [Qiskit](/docs/api/qiskit-ibm-runtime/ibm-backend).
+The first section of the QPU information card provides the following QPU details: 
 
-These values are shown on both the [Compute resources page](https://quantum.cloud.ibm.com/computers) and the details page for each QPU.
+* **Name** - The unique name assigned to a specific QPU. QPUs hosted on IBM Cloud® have names that begin with `ibm_*`. All QPUs are given a city name, e.g., `ibm_kingston`. This name does not indicate where the actual QPU is hosted. They are named after IBM&reg; locations around the world.
+* **Qubits** - The number of physical qubits in a QPU.
+* **2Q error (layered)** - Average error per layered gate (EPLG) in a chain of 100 qubits. Average EPLG measures the average gate error in a layered chain of $N$ qubits ($N$=100 here). It is derived from a similar quantity known as the layer fidelity (LF) where EPLG$_{100}$ = 4/5(1-LF$^{\frac{1}{99}}$) and layer fidelity is the process fidelity of the layered chain of $N$ qubits. For details, see the paper [Benchmarking quantum processor performance at scale](https://arxiv.org/abs/2311.05933). Note that in the paper EPLG is defined for process error, but for consistency with the individually reported gate errors here it is quoted for average gate error, thus the factor of 4/5.
+* **CLOPS** (or CLOPS_h) - Circuit layer operations per second, is a measure of how many layers of a 100x100 circuit (hardware-aware circuit) a QPU (quantum processing unit) can execute per unit of time.
```

Comment:
ADd a link to the CLOPS code here https://github.com/qiskit-community/qiskit-device-benchmarking/tree/main/qiskit_device_benchmarking/clops